### ISSUE TITLE
All Audio Devices Gone

### ISSUE THREAD

**Issue by darkMatterSound** [linuxmint/Cinnamon#4167]
Just ran an upgrade and all of my audio devices are now gone.  No panel indicator and no devices in the sound options in system settings.  Input or output.

**Comment by dalcde** [comment]
What did you upgrade from/to?

**Comment by darkMatterSound** [comment]
Upgraded to the latest version on deb http://ppa.launchpad.net/gwendal-lebihan-dev/cinnamon-nightly/ubuntu trusty main.  At around 8p PST 05/18.  Before that, I was running the next recent version from 05/17.  Everything was working fine with the version from 05/17.

**Comment by dalcde** [comment]
I don't see see any relevant changes in the codebase during that period. Did you upgrade anything else?

**Comment by darkMatterSound** [comment]
Hrm, not that I know of. Is there a way I can check what was updated for you?  I have a backup from earlier in the day. What files should I restore to get my audio working again?

**Comment by dalcde** [comment]
There should be some log files in /var/log that logs everything apt has done for you

**Comment by darkMatterSound** [comment]
Here's the relevant stuff from the apt log.  I excluded a lot of the other stuff I did, along with things earlier than when the problem started.  The pulseaudio stuff at the end is me trying to fix the issue.

Start-Date: 2015-05-18  18:26:04
Commandline: apt-get upgrade -y
Upgrade: python-nemo:amd64 (2.4.1-20150518023003-trusty, 2.4.1-20150519023003-trusty), cinnamon-settings-daemon:amd64 (2.5.0-20150518004503-trusty, 2.5.0-20150519004503-trusty), gir1.2-cmenu-3.0:amd64 (2.5.0-20150518012137-trusty, 2.5.0-20150519012140-trusty), nemo-data:amd64 (2.5.1-20150518010004-trusty, 2.5.1-20150519010004-trusty), cinnamon-bluetooth:amd64 (3.8.7-20150518012003-trusty, 3.8.7-20150519012002-trusty), libmuffin0:amd64 (2.5.0-20150518003003-trusty, 2.5.0-20150519003004-trusty), nemo:amd64 (2.5.1-20150518010004-trusty, 2.5.1-20150519010004-trusty), muffin:amd64 (2.5.0-20150518003003-trusty, 2.5.0-20150519003004-trusty), gir1.2-nemo-3.0:amd64 (2.5.1-20150518010004-trusty, 2.5.1-20150519010004-trusty), libnemo-extension1:amd64 (2.5.1-20150518010004-trusty, 2.5.1-20150519010004-trusty), gir1.2-muffin-3.0:amd64 (2.5.0-20150518003003-trusty, 2.5.0-20150519003004-trusty), nemo-fileroller:amd64 (2.4.0-20150518020003-trusty, 2.4.0-20150519020003-trusty), nemo-share:amd64 (2.4.0-20150518025056-trusty, 2.4.0-20150519025057-trusty), cinnamon-screensaver:amd64 (2.5.0-20150518013254-trusty, 2.5.0-20150519013251-trusty), cjs:amd64 (2.5.0-20150518020138-trusty, 2.5.0-20150519020135-trusty), libcinnamon-menu-3-0:amd64 (2.5.0-20150518012137-trusty, 2.5.0-20150519012140-trusty), libcinnamon-control-center1:amd64 (2.5.0-20150518013056-trusty, 2.5.0-20150519013054-trusty), cinnamon-control-center:amd64 (2.5.0-20150518013056-trusty, 2.5.0-20150519013054-trusty), muffin-common:amd64 (2.5.0-20150518003003-trusty, 2.5.0-20150519003004-trusty), libcjs0e:amd64 (2.5.0-20150518020138-trusty, 2.5.0-20150519020135-trusty), cinnamon-control-center-data:amd64 (2.5.0-20150518013056-trusty, 2.5.0-20150519013054-trusty)
End-Date: 2015-05-18  18:37:12

Start-Date: 2015-05-18  18:53:05
Commandline: apt-get upgrade -y
Upgrade: nemo-folder-color-switcher:amd64 (1.0.3-20150518032004-trusty, 1.0.3-20150519032008-trusty), folder-color-switcher:amd64 (1.0.3-20150518032004-trusty, 1.0.3-20150519032008-trusty), nemo-emblems:amd64 (2.4.3-20150518031505-trusty, 2.4.3-20150519031509-trusty)
End-Date: 2015-05-18  18:53:34

Start-Date: 2015-05-18  20:15:12
Commandline: apt-get upgrade
Upgrade: cinnamon-common:amd64 (2.5.0-20150518040006-trusty, 2.5.0-20150519040007-trusty), cinnamon-translations:amd64 (2.5.0-20150518040122-trusty, 2.5.0-20150519040127-trusty), cinnamon:amd64 (2.5.0-20150518040006-trusty, 2.5.0-20150519040007-trusty), mint-mdm-themes-gdm:amd64 (1.8.1-20150518040339-trusty, 1.8.1-20150519040450-trusty)
End-Date: 2015-05-18  20:15:50

Start-Date: 2015-05-18  21:26:37
Commandline: apt-get upgrade
Upgrade: cinnamon-session:amd64 (2.5.1-20150518043003-trusty, 2.5.1-20150519043003-trusty), cinnamon-session-common:amd64 (2.5.1-20150518043003-trusty, 2.5.1-20150519043003-trusty), mint-x-icons:amd64 (1.2.6-20150518044533-trusty, 1.2.6-20150519044542-trusty)
End-Date: 2015-05-18  21:27:00

Start-Date: 2015-05-18  21:36:30
Commandline: apt-get install pulseaudio indicator-sound indicator-sound-gtk2
Install: gsettings-ubuntu-schemas:amd64 (0.0.1+14.04.20140401-0ubuntu1, automatic), libido-0.1-0:amd64 (12.10.0.1-0ubuntu2, automatic), indicator-sound-gtk2:amd64 (12.10.0.1-1linuxmint2), indicator-sound:amd64 (12.10.2+14.04.20140401-1linuxmint1), liburl-dispatcher1:amd64 (0.1+14.04.20140403-0ubuntu1, automatic), libgee2:amd64 (0.6.8-1ubuntu1, automatic)
End-Date: 2015-05-18  21:36:41

Start-Date: 2015-05-18  21:36:52
Commandline: apt-get --reinstall install pulseaudio indicator-sound indicator-sound-gtk2
Reinstall: pulseaudio:amd64 (4.0-0ubuntu11.1), indicator-sound-gtk2:amd64 (12.10.0.1-1linuxmint2), indicator-sound:amd64 (12.10.2+14.04.20140401-1linuxmint1)
End-Date: 2015-05-18  21:37:01
